definitions/infra-vspherecluster/golden_metrics.yml

```diff
@@ -1,21 +1,30 @@
-effectiveCpuAvailableToVmsMhz:
-  title: Effective CPU available to VMs (MHz)
+overallStatus:
+  title: Overall Status of the Cluster
   query:
-    select: average(cpu.totalEffectiveMHz)
-    from: VSphereClusterSample
-    facet: entityName
     eventId: entityGuid
-effectiveMemoryAvailableToVmsMb:
-  title: Effective memory available to VMs (MB)
+    select: latest(`overallStatus`)
+    from: VSphereClusterSample
+effectiveHosts:
+  title: Effective Hosts in the Cluster
   query:
-    select: average(mem.effectiveSize)
+    eventId: entityGuid
```

Comment:
Being the eventId entityGuid instead of entity.guid should the facet also be eventName instead of entity.name ?